package.json

```diff
@@ -47,13 +50,13 @@
     "lru-cache": "~2.7.3",
     "morgan": "^1.9.1",
     "ms": "^2.0.0",
-    "nconf": "https://github.com/jfromaniello/nconf/tarball/master",
+    "multer": "^1.4.5-lts.1",
+    "nconf": "^0.12.0",
     "object-sizeof": "^1.4.0",
     "passport": "~0.1.16",
     "passport-local": "~0.1.6",
     "passport-ssl-certificate": "https://github.com/jaredhanson/passport-ssl-certificate.git#826c16d040841ec4b20db1a4cfe8bac81c931462",
     "passport-windowsauth": "^3.0.0",
-    "request": "^2.88.0",
```

Comment:
Any reason to update this as part of the node upgrade?

Reply:
This is being done as part of the `qs` sec fix part of the ticket.